### PR TITLE
[Hub Generated] Review request for Microsoft.Insights to add version preview/2020-02-10-preview

### PR DESCRIPTION
**Pull request by aarongas**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/aarongas/azure-rest-api-specs/dev-applicationinsights-Microsoft.Insights-2020-02-10-preview...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by sanjaiganesh — CHANGES_REQUESTED** [review]

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            "description": "*** Error Responses: ***\n\n * 400 Bad Request.\n\n * 500 Internal Server Error."
+          }
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```
[the remark this answers is not in the document]
Comment (aarongas) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json]: fixed.

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/webtests/{webTestName}/gettestresultfile": {
```
[the remark this answers is not in the document]
Comment (aarongas) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json]: fixed

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+      "post": {
+        "tags": [
+          "FindWebTestResult"
+        ],
+        "operationId": "gettestresultfile",
```
[the remark this answers is not in the document]
Comment (aarongas) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json]: fixed

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+      "enum": [
+        "webtestresult",
+        "json"
+      ],
+      "x-ms-enum": {
+        "name": "downloadAs",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "webtestresult",
```
[the remark this answers is not in the document]
Comment (aarongas) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json]: fixed

File: specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+        }
+      }
+    },
+    "TestResultFileResponse": {
+      "description": "Test result.",
+      "type": "object",
+      "properties": {
+        "data": {
+          "description": "File contents.",
+          "type": "string"
```
[the remark this answers is not in the document]
Comment (aarongas) [reply specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-02-10-preview/WebTestResults_API.json]: I would like to leave as string for now.

**Review by sanjaiganesh — APPROVED** [review]

**Review by zikalino — APPROVED** [review]

**Pull request merged**